libraries/AP_GPS/AP_GPS.h

```diff
@@ -391,10 +406,11 @@ class AP_GPS
         // the time we got our last fix in system milliseconds
         uint32_t last_message_time_ms;
     };
-    GPS_timing timing[GPS_MAX_INSTANCES];
-    GPS_State state[GPS_MAX_INSTANCES];
-    AP_GPS_Backend *drivers[GPS_MAX_INSTANCES];
-    AP_HAL::UARTDriver *_port[GPS_MAX_INSTANCES];
+    // Note allowance for an additional instance to contain blended data
+    GPS_timing timing[GPS_MAX_RECEIVERS+1];
+    GPS_State state[GPS_MAX_RECEIVERS+1];
```

Comment:
I find the +1 more clearer too, but then I think we should just scrape the GPS_MAX_INSTANCES entirely (or even better, use it instead of creating a new GPS_MAX_RECEIVERS). The vehicles should really use the *num_sensors* method instead of relying on using the macro anyway.